endtoend/evaluators/slashing.go

```diff
@@ -35,14 +35,14 @@ var ProposeDoubleBlock = types.Evaluator{
 // ValidatorsSlashed ensures the expected amount of validators are slashed.
 var ValidatorsSlashed = types.Evaluator{
 	Name:       "validators_slashed_epoch_%d",
-	Policy:     policies.AfterNthEpoch(1),
+	Policy:     policies.AfterNthEpoch(2),
```

Comment:
Why are there changes here?

Reply:
Sorry I was doing some local testing. Didn't mean to push this. Will revert it back